test/integration/testgrid-config-generator/config/_allow-list.yaml

```diff
@@ -0,0 +1,3 @@
+release-openshift-origin-job: informing
```

Comment:
I was expecting a new set of jobs in the allow-list to cause us to have new output from the tool. This test as written doesn't prove that your code works :)